### PR TITLE
update 3.26.4

### PR DESCRIPTION
**Pull request by Arishamays1**
## Cmake 3.26.4 Update
[Jira Ticket](https://anaconda.atlassian.net/browse/PKG-2183)
[Upstream](https://cmake.org/)
# Changes
- Updated version number and `sha256` in `meta.yaml` and `ssltest.cmake`
- Updated pinnings on dependencies
- Removing outdated patch
- Fixed `license urls`
- Updated `cbc.yaml` for compiler support for `win64`       
- Amendments made to `bld.bat`, as `JOM` requests for cmake` 
to generate JOM files on Windows. Generating regular `NMake ` files instead of `NMake jom`.

### CODE REVIEW

**Review by JeanChristopheMorinPerso — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,19 +21,31 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - make            # [unix]
-    - m2-gcc-libs     # [win]
-    - jom             # [win]
+    - patch           # [not win]
+    - m2-patch        # [win]
```
[the remark this answers is not in the document]
Comment (Arishamays1) [reply recipe/meta.yaml]: removed

File: recipe/bld.bat [hunk 2]
```diff
@@ -25,7 +25,11 @@ cmake -LAH -G"NMake Makefiles JOM"                           ^
     -DCMAKE_USE_SYSTEM_LIBLZMA=TRUE                          ^
     -DCMAKE_USE_SYSTEM_ZLIB=TRUE                             ^
     -DCMAKE_USE_SYSTEM_BZIP2=TRUE                            ^
-    -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" ..\cmake
+    -DZLIB_LIBRARY="%LIBRARY_LIB%/zlib.lib"                  ^
```
Comment (marcoesters) [inline recipe/bld.bat:28]:
```suggestion
    -DZLIB_LIBRARY="%LIBRARY_LIB%\zlib.lib"                  ^
```

**Review by marcoesters — APPROVED** [review]
Just a minor cosmetic suggestion - doesn't seem to prevent compilation though

File: recipe/meta.yaml [hunk 3]
```diff
@@ -26,19 +21,29 @@ requirements:
...
+    - expat           # [unix]
+    - ncurses         # [unix]
     - xz
     - zlib
-    - zstd
     - libuv
-
+    - rhash           # [unix]
+    - zstd
+    - vs2015_runtime  # [win]
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:46]: I'm curious, we specify vs2019 in cbc. Why is this 2015?

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,23 +1,18 @@
...
   - url: https://cmake.org/files/v{{ version[:4] }}/cmake-{{ version }}-windows-x86_64.zip  # [win64]
-    sha256: 35fbbb7d9ffa491834bbc79cdfefc6c360088a3c9bf55c29d111a5afa04cdca3  # [win64]
+    sha256: 62c35427104a4f8205226f72708d71334bd36a72cf72c60d0e3a766d71dcc78a  # [win64]
     folder: cmake-bin  # [win]
-
 build:
   number: 0
-  skip: True  # [win and vc<14]
-  detect_binary_files_with_prefix: False
+  skip: True  # [py<38]
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:15]:
```suggestion
```
CMake doesn't depend or require python.

**Review by JeanChristopheMorinPerso — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 3 above)
Comment (ELundby45) [reply recipe/meta.yaml:46]: @Arishamays1 I added the cbc on my path of figuring out the windows issues. VS2019 shouldn't be necessary, you can remove the `conda_build_config.yaml` file.

Comment (Arishamays1) [reply recipe/meta.yaml:46]: Got it! Removed. Thanks for your help Eric!

File: recipe/bld.bat (hunk 2 above)
Comment (Arishamays1) [reply recipe/bld.bat:28]: Thanks! Made this change

File: recipe/meta.yaml (hunk 4 above)
Comment (Arishamays1) [reply recipe/meta.yaml:15]: Ah you're right. Removed. Thanks JC!

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Pull request merged**